RevenueCatUI/Views/FooterView.swift

```diff
@@ -27,6 +27,7 @@ struct FooterView: View {
     var boldPreferred: Bool
     var purchaseHandler: PurchaseHandler
     var displayingAllPlans: Binding<Bool>?
+    static let buttonMinHeight: CGFloat = 44
```

Comment:
This is a great idea 👏🏻 

Suggestion:
- Let's extract this to `Constants.swift`
- We can define it like this to be consistent with the [design guidelines](https://developer.apple.com/design/human-interface-guidelines/buttons#Best-practices) (maybe even add a comment to reference it):

```swift
// See https://developer.apple.com/design/human-interface-guidelines/buttons#Best-practices
#if swift(>=5.9) && os(visionOS)
static let minimumButtonHeight: CGFloat = 60
#else
static let minimumButtonHeight: CGFloat = 44
#endif
```